packages/snjs/lib/services/search_service.ts

```diff
@@ -0,0 +1,58 @@
+import { SNNote } from "@Lib/models/app/note";
```

Comment:
So sometime before the 3.6 release and after you started working on this we updated this file on web to not search protected note bodies, and I think you'll want to add these changes in!

Reply:
Did.